src/index.ts

```diff
@@ -1,11 +1,16 @@
 import { Union as _Union } from './union';
 import { IFS } from './fs';
+export * from './lists'
 
 export interface IUnionFs extends IFS {
-  use(fs: IFS): this;
+  use: (...args: Parameters<_Union['use']>) => this;
```

Comment:
I don't expect the signature of this to change too much, so think it's fine to inline here for readability.